cupy/fft/config.py

```diff
@@ -9,9 +9,12 @@
 from cupy.fft._cache import set_plan_cache_max_memsize  # NOQA
 from cupy.fft._cache import show_plan_cache_info  # NOQA
 
-# expose callback handles to this module
-from cupy.fft._callback import get_current_callback_manager  # NOQA
-from cupy.fft._callback import set_cufft_callbacks  # NOQA
+# on Linux, expose callback handles to this module
+import sys
+if sys.platform.startswith('linux'):
```

Comment:
I wonder if this will work with WSL?

Reply:
This is defined the same in https://github.com/cupy/cupy/blob/master/install/build.py#L14 so it should work fine.

Reply:
> I wonder if this will work with WSL?

Good question, but I do not know 😂 @mnicely @llukas Does cuFFT callback work with WSL?